### DISCUSSION TITLE
URGENT : Including the blocks in PHP Version Editor

### DISCUSSION THREAD

**Discussion by mobiliseapplabllp** [givanz/VvvebJs#384] [Q&A]
Dear Givaz,
 
I hope you're doing well.
We are currently using the PHP version of the Vvveb editor, which we installed from GitHub. However, we have encountered an issue where certain blocks available in the demo version, accessible at the link below, are missing from our setup:
 
https://demo.vvveb.com/admin/?module=/editor/editor&template=index.html
 
Image
 
As these blocks are crucial for our project, we would greatly appreciate your guidance on how to import or enable them in our version. We have already integrated this builder deeply into our system, so resolving this issue is essential for our progress.
 
Your support would be invaluable, and we are eager to hear any suggestions or solutions you may have.
 
Thank you in advance for your time and assistance. We are hoping for your prompt help on this matter.
 
Warm regards,
Ashish Sharma

<img width="903" alt="Screenshot 2024-10-11 at 15 05 24" src="https://github.com/user-attachments/assets/e1846c34-e661-44e9-8d3a-c42a0fd1bc96">

**Comment by givanz** [comment]
https://github.com/givanz/VvvebJs/issues/385 (duplicate)
